elf/elf_loader.c

```diff
@@ -30,14 +30,11 @@
 
 #include "elf_loader.h"
 #include "../dbm.h"
-#include "../common.h"
 
 #ifndef AT_MINSIGSTKSZ
   #define AT_MINSIGSTKSZ 51
 #endif
 
-#define DEBUG 1
```

Comment:
Don't set the `DEBUG` flag

Reply:
Here I am lost. The original code is:

```C
#define DEBUG 1
#undef DEBUG
```

The commit is removing both lines, as I think they cancel each other.
What change are you proposing here?